llap-server/src/java/org/apache/hadoop/hive/llap/io/api/impl/LlapIoImpl.java

```diff
@@ -112,6 +115,8 @@
   private final BufferUsageManager bufferManager;
   private final Configuration daemonConf;
   private final LowLevelCacheMemoryManager memoryManager;
+  private LowLevelCachePolicy realCachePolicy;
```

Comment:
Why was this refactor necessary? (similarly why for tracePool)

Reply:
Both will be required in HIVE-24727, I've reverted them for now.